### PR TITLE
Fix CC license CSP error

### LINKED ISSUE #14889: CSP img error on /foundation/licensing/website-content/
## Description

The Creative Commons badge on the /foundation/licensing/website-content/ page does not load due to a CSP violation.

### Steps to reproduce

1. https://www.mozilla.org/en-US/foundation/licensing/website-content/

### Expected result

The Creative Commons [badge](https://mirrors.creativecommons.org/images/public/somerights20.gif) should be displayed.

### Actual result

The image is blocked due to a CSP violation. It looks like the image URL now forwards to `mirrors.creativecommons.org`.

We can probably just load this image directly from `/media`, rather than need to make a CSP exception for a third party domain for just a single image.

```
content-Security-Policy: The page’s settings blocked the loading of a resource (img-src) at https://mirrors.creativecommons.org/images/public/somerights20.gif because it violates the following directive: “img-src https://www.google-analytics.com/ https://www.googletagmanager.com/ https://images.ctfassets.net/ https://*.mozilla.org data: https://*.mozilla.com https://*.mozilla.net 'self' https://creativecommons.org/ https://mozilla.org/”
```

### Environment

Firefox Nightly
macOS

### PR DESCRIPTION
**Pull request by janbrasna**
## One-line summary

CC serves their assets via mirror* subdomain, violating the CSP rule here. Keeping the policy loosened for just this one occurrence doesn't seem to be worth it so this PR stores the SVG badge for CC-BY-SA locally.

## Significant changes and points to review

Also removes `creativecommons.org` from `img-src` CSP.

## Issue / Bugzilla link

Fixes #14889

## Testing

http://localhost:8000/en-US/foundation/licensing/website-content/
~~(+to see the additional report-only header locally, you have to provide some reporting endpoint in ENV for [`CSP_REPORT_URI`](https://github.com/mozilla/bedrock/blob/d6c387188df907fa2793d3ff9979a47353a1895e/app.json#L15))~~

### CODE REVIEW

File: bedrock/foundation/templates/foundation/licensing/website-content.html
Comment (alexgibson) [inline bedrock/foundation/templates/foundation/licensing/website-content.html:22]: Can we please just load this as a regular ``? I don't think we really gain much here by in-lining the image, and having inline images makes them hard to search for in the future.

File: bedrock/settings/__init__.py
Comment (alexgibson) [inline bedrock/settings/__init__.py:273]: I think we are pretty safe just to remove `creativecommons.org` from `_csp_img_src` entirely, as this seems to be the only image we load from that domain.

**Review by alexgibson — CHANGES_REQUESTED** [review]
Thanks for helping out with this! Just a couple of things below:

File: bedrock/foundation/templates/foundation/licensing/website-content.html
Comment (janbrasna) [reply bedrock/foundation/templates/foundation/licensing/website-content.html:22]: I originally used a more modern iconography, that was just a few shapes:

> Screenshot 2024-07-29 at 11 07 44

But they seem not to use the "some rights reserved" tagline much these days anymore, so I opted for one of the buttons instead eventually.

Looking at this button code, it's rather large, yea. I'll move it out.

**Review by alexgibson — APPROVED** [review]
r+ thanks!

I'm going to hold off merging this as we're now in a code freeze for our upcoming company moz-week event. But we'll merge this after Aug 19th.

**Comment by janbrasna** [comment]
Ack. Did my best to hotfix it beforehand;) to silence the logs/reports, but given this page is outside of any IA structure it hopefully won't get much pings during the few weeks.

**Comment by alexgibson** [comment]
> but given this page is outside of any IA structure it hopefully won't get much pings during the few weeks.

Yeah this page gets very little (to virtually no) traffic. It's likely content that we will remove/migrate somewhere else in the mid-term future as part of the site refresh.

**Comment by alexgibson** [comment]
Going to merge this (we discussed thing like this are OK to merge, but we won't deploy them)

**Pull request merged**
